docs/api/modules/main/exports/schema.md

```diff
@@ -428,7 +428,7 @@ todo
 import { schema } from 'nexus'
 import { connectionFromArray } from 'graphql-relay'
 
-schema.queryType(t => {
+schema.queryType((t) => {
```

Comment:
Unrelated to the PR but that thing looks wrong to me, it should be in a `definition` block